portfolio/src/main/webapp/index.html

```diff
@@ -1,16 +1,32 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-    </div>
-  </body>
-</html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div id="fullpage">
+		<div class="section">
+		</div>
+		<div class="section">
+		</div>
+        <div class="section" id="projects">
+            <a class="card">InfluencerAds</a>
+            <a class="card">Permutation Palette Picker</a>
+            <a class="card">Auction Search Tool</a>
+            <a class="card">Genesis Trust Volunteer Portal</a>
+            <a class="card">2048 Clone</a>
+            <a class="card">Something</a>
+		</div>
```

Comment:
the indentation of the closing `</div>` should match the indentation of the corresponding opening `<div>` tag, here and elsewhere.